packages/jobs/lib/workflows.ts

```diff
@@ -19,6 +19,11 @@ export async function initialSync(args: InitialSyncArgs): Promise<boolean | obje
     try {
         return await routeSync(args);
     } catch (e: any) {
+        if (isCancellation(e)) {
+            await CancellationScope.nonCancellable(() => cancelActivity(args));
+
+            return false;
+        }
```

Comment:
What happen if that call fails? (not sure it can really just asking)

Reply:
It would just not cancel correctly and the workflow would continue.

Reply:
yeah sorry I miss spoke, I meant since it's not in a try/catch will the error bubble and crashes (or something else)?